tests/AbstractTestCase.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Psr\Log\Util\Tests;
+
+if (class_exists('PHPUnit\Framework\TestCase')) {
```

Comment:
Please use `TestCase::class` et al any time you reference a class, not a string.  That's valid since PHP 5.5 and makes the code much more maintainable and readable.

Reply:
Test classes are not part of the spec, so they can be whatever the heck we want. :smile: 

The stats from Packagist show that PHP < 5.5 installs are barely 2%, and PHP 8 is 7 times as widely used: https://packagist.org/packages/psr/log/php-stats

So I officially do not give a damn about PHP versions 5.4 and lower.

Reply:
> Test classes are not part of the spec, so they can be whatever the heck we want.

Not part of the spec, but the spec says there's support for 5.3 and 5.4, and strictly speaking we must build on those versions too in order to ascertain that this package actually complies with the spec.

> So I officially do not give a damn about PHP versions 5.4 and lower.

Fair enough. Does there need to be some consensus on this, or should I just do it? Sorry, I don't really know how things work between you 😅

Reply:
Since this is aimed to replace the stripped classes from `psr/log`, I would support 5.3 but drop it immediately after with another release.

Reply:
That's what I'd also do, if my understanding of the goals of this first release is correct ☝️